src/main/java/uk/ac/ebi/subs/fileupload/errors/ErrorMessages.java

```diff
@@ -5,7 +5,8 @@
  */
 public class ErrorMessages {
 
-    public static final String INVALID_PARAMETERS = "Invalid parameters";
-    public static final String NOT_SUPPORTED_EVENT = "Not supported event";
-    public static final String FILE_DOCUMENT_NOT_FOUND = "File document is not found in the database";
+    public static final String INVALID_PARAMETERS = "Invalid parameters.";
+    public static final String NOT_SUPPORTED_EVENT = "Not supported event.";
+    public static final String FILE_DOCUMENT_NOT_FOUND = "File document is not found in the database.";
+    public static final String FILE_IO_ERROR = "File could not be created.";
```

Comment:
Given the error text, should this variable be called FILE_CREATION_ERROR ?

Reply:
Agree